src/pages/List.js

```diff
@@ -221,26 +223,30 @@ export default function List({ token }) {
       <h1>This Is Your Grocery List</h1>
       <h2>It uses the token: {token}</h2>
       <label htmlFor="thesearch">Search Grocery List Items </label>
-      <input
-        type="text"
-        placeholder="enter grocery item"
-        value={query}
-        id="thesearch"
-        onChange={(e) => setQuery(e.target.value)}
-      />
-      <button onClick={handleReset}>Reset Text Field</button>
+      <div className="flex">
+        <input
+          type="text"
+          placeholder="enter grocery item"
+          value={query}
+          id="thesearch"
+          onChange={(e) => setQuery(e.target.value)}
+        />
+        <IconButton onClick={handleReset} text="X" />
```

Comment:
What would a  screen reader say for this button?

Reply:
Well, I'm not TOO sure, but it is just a <button> that's being imported. BUT this is a great point, and I think I will have to look  at the accessibility of the button and make sure it's labeled correctly!

Reply:
Ok, I found a [GREAT SOLUTION](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA24.html) for wrapping an icon in a `<span>` and giving it a `role="img"` and `aria-label={label}` props.

```javascript 
<button
      onClick={onClick}
      className={`bg-gray-200 text-midnight-green px-2 rounded font-medium py-0 ml-2 hover:bg-caribbean-green ${tailwind}`}
    >
      <span role="img" aria-label={label}>
        {icon}
      </span>
</button>
```

Reply:
And then pass the text of the `label` that we want read by a screen reader as such:

```javascript
<IconButton type="submit" icon=">" label="submit token" />
```